### ISSUE TITLE
SyntasticReset doesn't play well in custom command

### ISSUE THREAD

**Issue by knownasilya** [scrooloose/syntastic#1593]
So I'm attempting the following in my `~/.vimrc`:

```
command HideGutter set nonumber|GitGutterDisable|SyntasticReset
```

But if `SyntasticReset` is in the list, I get the following error:

> E488: Trailing characters: GitGutterDisable|SyntasticReset

If I remove it, everything works fine.. What could be the issue?

**Comment by lcd047** [comment]
Why do you think this has anything to do with syntastic?  As far as I can tell it's a bug in Vim.  Replace `GitGutterDisable` with its definition top get rid of the error, and report the problem to `vim_dev`.
```vim
command! HideGutter set nonumber|call gitgutter#disable()|SyntasticReset
```

**Comment by knownasilya** [comment]
What's `vim_dev`?

**Comment by lcd047** [comment]
First hit on Google: [vim_dev](https://groups.google.com/d/forum/vim_dev).

**Comment by lcd047** [comment]
Actually, I was wrong: this has to do with both syntastic and `vim-gitgutter` [not passing `-bar`](https://groups.google.com/forum/#!topic/vim_dev/hr1GY0MkbUk) to `command`.  Syntastic should be fixed in d79d77f.  A similar fix is needed for `vim-gitgutter`, but moving `GitGutterDisable` should now work:
```vim
command! HideGutter set nonumber|SyntasticReset|GitGutterDisable
```

**Comment by knownasilya** [comment]
Thanks for looking into it. The `command` documentation alone confuses me..
